### DISCUSSION TITLE
Add config options for external DB and Redis services

### DISCUSSION THREAD

**Discussion by mark-webster-catalyst** [nextcloud/all-in-one#4551] [Ideas] [answered]
It would be nice to have the option to point the installation at external database and redis services, (RDS and Elasticache, for example). This along with switching cron on/off via an env var would be a big step towards getting AIO deployable in ECS

**Comment by szaimen** [comment, marked as answer]
Hi, for "normal" AIO and the helm chart, this is out of scope. However you might make it work with https://github.com/nextcloud/all-in-one/tree/main/manual-install
